src/controller.py

```diff
@@ -13,6 +13,8 @@ def extract_args():
     parser.add_argument('--projects-title-filter', required=True, help='Filter projects by titles. Provide a list of project titles.')
     parser.add_argument('--milestones-as-chapters', required=True, help='Treat milestones as chapters in the generated documentation.')
     parser.add_argument('--repositories', required=True, help='JSON string defining the repositories to be included in the documentation generation.')
+    parser.add_argument('--data-directory', type=str, required=True, help='Data directory, where are generated all fetched and cosolidated data.')
```

Comment:
I can't remember, why did we put these parameters into controller script. It seems to work just fine also without them. We don't use them anymore in any of our scripts. It might be left out code after debugging.

Reply:
Solved in commit: [fac0445](https://github.com/AbsaOSS/living_doc_generator/pull/16/commits/fac0445143f226096fe31a80dec61f1fa523699a)